src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -156,7 +156,7 @@ IntPtr overlapped
             );
 
             // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
> The implementation currently has OS specific calls and so for now it can't be AnyCPU really. However depending on how #4786 is fixed it might change :). I will leave the right fix to you guys :)

Again, we handle the case where the P/Invoke entry point is not found. Many libs in the ecosystem do this too. The issue isn't a problem for non-Windows platforms. Satisfying WACK is the only problem here.